src/packages/OneUITheme/baseVariables/headerFooter.ts

```diff
@@ -0,0 +1,14 @@
+export const headerFooterVariables = {
+    // Header variables
+    '--header-background-color': 'var(--color-background) !default',
```

Comment:
I don't think we need `!default` here (and in all the cases below) since it's a SASS thing.